File: src/main/java/org/dxworks/githubminer/dto/response/repository/refs/Ref.java
package org.dxworks.githubminer.dto.response.repository.refs;

import com.google.api.client.json.GenericJson;
import com.google.api.client.util.Key;
import lombok.Data;

@Data
public class Ref extends GenericJson {
    @Key
    private String ref;
    @Key
    private RefObject object;
}

File: github-miner-core/src/main/kotlin/org/dxworks/githubminer/dto/response/repository/refs/Ref.kt
package org.dxworks.githubminer.dto.response.repository.refs

import com.google.api.client.json.GenericJson
import com.google.api.client.util.Key

class Ref : GenericJson() {
    @Key
    var ref: String? = null

    @Key
    var `object`: RefObject? = null
}
